File: GestionAeropuerto.API/Models/Pasajero.cs
namespace GestionAeropuerto.API.Models
{
    public class Pasajero
    {
        public int PasajeroID { get; set; }
        public string TipoDocumento { get; set; } = "CC";
        public string NumeroDocumento { get; set; } = string.Empty;
        public string Nombre { get; set; } = string.Empty;
        public string Apellido { get; set; } = string.Empty;
        public DateTime FechaNacimiento { get; set; }
        public string Nacionalidad { get; set; } = string.Empty;
        public string? Email { get; set; }
        public string? Telefono { get; set; }
        public DateTime FechaRegistro { get; set; }
    }
}

File: GestionAeropuerto.VB/Pasajero.vb
Public Class Pasajero
    Public Property PasajeroID As Integer
    Public Property TipoDocumento As String
    Public Property NumeroDocumento As String
    Public Property Nombre As String
    Public Property Apellido As String
    Public Property FechaNacimiento As DateTime
    Public Property Nacionalidad As String
    Public Property Email As String
    Public Property Telefono As String

    Public ReadOnly Property NombreCompleto As String
        Get
            Return Nombre & " " & Apellido
        End Get
    End Property

    Public Sub New()
        TipoDocumento = "CC"
        NumeroDocumento = ""
        Nombre = ""
        Apellido = ""
        Nacionalidad = "Colombiana"
        Email = ""
        Telefono = ""
    End Sub
End Class